jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/CaseServiceImpl.java

```diff
@@ -662,5 +724,6 @@ protected boolean isEmpty(Collection<?> data) {
         return false;
     }
 
+   
```

Comment:
If this line is not necessary, can it be removed?

Reply:
removed